test/test/test.js

```diff
@@ -81,14 +90,16 @@ describe('Bridge', function () {
       let amount = BigNumber('1000');
 
       let beforeEthBalance = await ethClient.getErc20Balance(ethClient.accounts[0], this.tokenAddress);
-      let beforeSubBalance = await subClient.queryAccountBalance(polkadotRecipientSS58, this.tokenAddress);
+      let beforeSubBalance = await subClient.queryAccountBalance(polkadotRecipientSS58, TKN_ASSET_ID);
 
       await ethClient.approveERC20(ethClient.accounts[0], amount, this.tokenAddress);
       await ethClient.sendERC20(ethClient.accounts[0], amount, this.tokenAddress, polkadotRecipient);
-      await sleep(10000);
+      await sleep(30000);
+			// await subClient.waitNewBlock();
```

Comment:
here and later in this file:
remove unused code

Reply:
Will do.